src/Orc.NuGetExplorer.Xaml/Messaging/ActivatedExplorerTabMessage.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Orc.NuGetExplorer.Messaging
```

Comment:
Maybe remove the .Messaging namespace?

Reply:
Depends on should we we use specific namespaces for grouped elements in subfolders (like service, packaging, management..) or not? What is our default convention?

Reply:
Overall the orc components should all be in the same namespace. We consider them "feature containers". Once you include Orc.NuGetExplorer, it should have all features available. Only very specific, internal, things might go into their own namespace (but better make them private or internal).